src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -377,7 +405,10 @@ private static function addExtra(array &$a, \Reflector $c)
         }
     }
 
-    private static function addMap(array &$a, \Reflector $c, array $map, string $prefix = Caster::PREFIX_VIRTUAL)
+    /**
+     * @param \Reflector|\ReflectionAttribute $c
```

Comment:
I think we should remove this one: we don't use any method from neither Reflector nor ReflectionAttribute. `object` is just correct here.

Reply:
All right, removed.